modules/gtdbtk/classifywf/main.nf

```diff
@@ -0,0 +1,86 @@
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options    = initOptions(params.options)
+
+
+def VERSION = '1.5.0' // When using stubs for the GTDB database, the version info isn't printed.
+
+process GTDBTK_CLASSIFYWF {
+    tag "${meta.assembler}-${meta.id}"
+
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['assembler', 'id']) }
```

Comment:
```suggestion
        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
```

Reply:
Ah, I see. Thanks for pointing this out - applying now.

Then, I presume that only the `id` field is considered during the publishing process?